src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/workflow.html.twig

```diff
@@ -23,6 +120,87 @@
                 flowchart: { useMaxWidth: false },
                 securityLevel: 'loose',
             });
+
+            {% for name, data in collector.workflows %}
+                window.showNodeDetails{{ collector.hash(name) }} = function (node) {
+                    const map = {{ data.listeners|json_encode|raw }};
+                    showNodeDetails(node, map);
+                };
+            {% endfor %}
+
+            const showNodeDetails = function (node, map) {
+                const dialog = document.getElementById('detailsDialog');
+
+                dialog.querySelector('tbody').innerHTML = '';
+                for (const [eventName, listeners] of Object.entries(map[node])) {
+                    listeners.forEach(listener => {
+                        const row = document.createElement('tr');
+
+                        const eventNameCode = document.createElement('code');
+                        eventNameCode.textContent = eventName;
+
+                        const eventNameCell = document.createElement('td');
+                        eventNameCell.appendChild(eventNameCode);
+                        row.appendChild(eventNameCell);
+
+                        const listenerDetailsCell = document.createElement('td');
+                        row.appendChild(listenerDetailsCell);
+
+                        let listenerDetails;
+                        const listenerDetailsCode = document.createElement('code');
+                        listenerDetailsCode.textContent = listener.title;
+                        if (listener.file) {
+                            const link = document.createElement('a');
+                            link.href = listener.file;
+                            link.appendChild(listenerDetailsCode);
+                            listenerDetails = link;
+                        } else {
+                            listenerDetails = listenerDetailsCode;
+                        }
+                        listenerDetailsCell.appendChild(listenerDetails);
+
+                        if (typeof listener.guardExpressions === 'object') {
+                            listenerDetailsCell.appendChild(document.createElement('br'));
+
+                            const guardExpressionsWrapper = document.createElement('span');
+                            guardExpressionsWrapper.appendChild(document.createTextNode('guard expressions: '));
+
+                            const guardExpressionsCode = document.createElement('code');
+                            guardExpressionsCode.textContent = listener.guardExpressions.join('</code>, <code>');
```

Comment:
this is broken. this will create a single code element with `foo</code>, <code>bar` as textual content. you need to create one `code` element per guard expression and add `, ` text nodes between them.

Reply:
Thanks, fixed